docs/apps/crypto-trading.md

```diff
@@ -3,178 +3,197 @@ sidebar_position: 2
 title: Crypto Trading Bot
 ---
 
-## On GDN
+## Crypto App details
 
 | **Tenant** | **Fabric** | **Stream App** | **GUI** | **Source Code**|
 |----------- |----------|----|-----------|-----------|
 |  `demo@macrometa.io` | `_system` | `crypto-trading-app` | [**Crypto Trading GUI**](https://macrometacorp.github.io/tutorial-cryptotrading/) |[github](https://github.com/Macrometacorp/tutorial-cryptotrading)|
 
-## Stream App
+Read our [Blog Post](https://www.macrometa.com/blog/cryptocurrency-trading-building-a-multi-exchange-global-trading-bot) about exchange arbitrage and how the app works.
```

Comment:
```suggestion
Read our blog post [How To Build A Crypto Arbitrage Trading Bot](https://www.macrometa.com/blog/cryptocurrency-trading-building-a-multi-exchange-global-trading-bot) to learn more about exchange arbitrage and how the app works.
```

Reply:
"Blog post" is extremely vague link text. Try to be as specific as possible, preferably matching the title of whatever you are linking to.